components/portal/src/components/appLayout/footer.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import CelleryLogo from '../../img/celleryLogo.svg';
+import Container from '@material-ui/core/Container';
+import Link from '@material-ui/core/Link';
+import React from "react";
+import Typography from '@material-ui/core/Typography';
+import {withStyles} from "@material-ui/core/styles";
+import * as PropTypes from "prop-types";
+
+const styles = (theme) => ({
+    footerContent: {
+        backgroundColor: '#e5eaea',
+        color: '#57595d',
+        height: 60,
+        display: "flex",
+        alignItems: "center"
+    },
+    footerLink: {
+        textTransform: 'uppercase',
+        color: '#57595d',
+        letterSpacing: 1,
+        fontWeight: 500,
+        textDecoration: 'none',
+    },
+    title: {
+        flexGrow: 1,
+    },
+    copyRightInfo: {
+        display: 'flex'
+    },
+    celleryLogoFooter: {
+        height: 20,
+        verticalAlign: 'middle',
+    }
+});
+
+class AppLayout extends React.Component {
```

Comment:
Shouldn't the name of this component be Footer ? Considering the content and filename.